workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -84,8 +83,20 @@ def add_tendency(state: Any, tendency: State, dt: float) -> State:
 
     with xr.set_options(keep_attrs=True):
         updated = {}
-        for name in tendency:
-            state_name = TENDENCY_TO_STATE_NAME.get(name, name)
```

Comment:
Previously, the nudging stepper would return tendencies with the same name as the state variable.